Remove `ember-cli-string-helpers` addon

When I upgrade my addon to use `@ember/string` v4.0, my addon build will throw an error that says:

"Module not found: Error: ember-cli-string-helpers is trying to import the app's @ember/string package, but it seems to be missing"

I noticed that the `ember-cli-string-helpers` addon was archived in October 2024. I think it would be a good idea to remove this package and replace it with something that is maintained. For now, the way to fix this issue is by locking the version of `@ember/string` to 3.1.1.

--- a/addon/helpers/capitalize.js
+++ b/addon/helpers/capitalize.js
@@ -1,0 +1,6 @@
+import { helper } from '@ember/component/helper';
+import { capitalize } from '@ember/string';
+
+export default helper(function capitalizeHelper(positional) {
+  return capitalize(positional[0]);
+});

--- a/app/helpers/capitalize.js
+++ b/app/helpers/capitalize.js
@@ -1,0 +1,1 @@
+export { default } from 'ember-cli-addon-docs/helpers/capitalize';

--- a/package.json
+++ b/package.json
@@ -53,7 +53,6 @@
     "ember-cli-clipboard": "^1.2.1",
     "ember-cli-htmlbars": "^6.3.0",
     "ember-cli-postcss": "^8.2.0",
-    "ember-cli-string-helpers": "^6.1.0",
     "ember-cli-string-utils": "^1.1.0",
     "ember-cli-version-checker": "^5.1.2",
     "ember-code-snippet": "^3.0.0",

--- a/tests/dummy/config/environment.js
+++ b/tests/dummy/config/environment.js
@@ -39,6 +39,10 @@ module.exports = function (environment) {
 
     ENV.APP.rootElement = '#ember-testing';
     ENV.APP.autoboot = false;
+
+    ENV['ember-tether'] = {
+      bodyElementId: 'ember-testing',
+    };
   }
 
   if (environment === 'production') {

--- a/tests/integration/helpers/capitalize-test.js
+++ b/tests/integration/helpers/capitalize-test.js
@@ -1,0 +1,18 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'dummy/tests/helpers';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import { capitalize } from '@ember/string';
+
+module('Integration | Helper | capitalize', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function (assert) {
+    const testString = 'abc 123 ABC !@# Foo Bar';
+    this.set('inputValue', testString);
+
+    await render(hbs`{{capitalize this.inputValue}}`);
+
+    assert.dom().hasText(capitalize(testString));
+  });
+});

